src/components/checkbox/checkbox.tsx

```diff
@@ -1,46 +1,29 @@
 import React = require('react');
-
-const style = require('./checkbox.st.css').default;
+import style from './checkbox.st.css';
+import { SBComponent } from 'stylable-react-component';
 
 export interface CheckBoxProps {
-    text: string;
     value: boolean;
     boxIcon: React.SFC<CheckBoxIconProps>;
```

Comment:
I wouldn't limit the icon properties to SFC. It should include ComponentClass as well. This means that it should be used differently in the render (createElement, not direct call)